Client/InfluxDBClient.cs

```diff
@@ -233,7 +233,96 @@ public class InfluxDBClient : AbstractRestClient, IInfluxDBClient
 
         private readonly Subject<Unit> _disposeNotification = new Subject<Unit>();
 
-        protected internal InfluxDBClient(InfluxDBClientOptions options)
+        /// <summary>
+        /// Create a instance of the InfluxDB 2.x client that is configured via <code>App.config</code>.
+        /// </summary>
+        public InfluxDBClient() :
+            this(InfluxDBClientOptions.Builder
+                .CreateNew()
+                .LoadConfig()
+                .Build())
+        {
+        }
```

Comment:
I prefer to remove this constructor without parameters, because the configuration via `App.config` is not common and `InfluxDBClient()` will not work in lot of cases.

```suggestion
```